Automattic/pocket-casts-ios
# New icon for prototype builds

Fixes #

This PR changes the icon for prototype builds to make easier to distinguish from the normal build

## To test

 - Install the prototype build using the QR code on this PR
 - Check that the icon is the electric blue one.
 - Install a debug build using the code in this branch and check that you get the original PocketCast icon

## Checklist
